### PR TITLE
change limited text area component to hooks implementation

### PR DESCRIPTION
**Pull request by phaniteja1**
I've changed the implementation of LimitedTextArea component to use React Hooks. I've also created and linked a sandbox snippet to demo the same --> https://codesandbox.io/s/5z2j52zvon

#36

### CODE REVIEW

File: snippets/LimitedTextarea.md [hunk 1]
```diff
@@ -2,42 +2,47 @@
...
+        value={content}
+      />
+      <p>
+        {content.length}/{limit}
+      </p>
+    </div>
+  );
 }
+
+export default LimitedTextArea;
```
Comment (Chalarangelo) [inline snippets/LimitedTextarea.md]: You should delete this line.

File: snippets/LimitedTextarea.md [hunk 2]
```diff
@@ -46,7 +51,7 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
-
+Sandbox Link: https://codesandbox.io/s/5z2j52zvon
```
Comment (Chalarangelo) [inline snippets/LimitedTextarea.md]: No links to external websites should be added, hence you should delete this, too. We will eventually set up a website where you can see all snippets in action.

File: snippets/LimitedTextarea.md (hunk 1 above)
Comment (phaniteja1) [reply snippets/LimitedTextarea.md]: Should I remove the top import line as well?

Comment (Chalarangelo) [reply snippets/LimitedTextarea.md]: Indeed you should, I kinda missed that one!

**Comment by phaniteja1** [comment]
@Chalarangelo I just opened another PR for the same #46 Can you please take a look at that. Thanks!

**Pull request closed without merging**